src/components/dialogs/limits/operational-limits-group-tab-label.tsx

```diff
@@ -45,9 +42,19 @@ export function OperationalLimitsGroupTabLabel({
         >
             <Stack direction="row" spacing={1} sx={{ alignItems: 'center' }}>
                 <Stack spacing={0}>
-                    <Typography color={permanentLimitErrorMessage ? red[500] : undefined}>
-                        {operationalLimitsGroup.name}
-                    </Typography>
+                    <Tooltip title={operationalLimitsGroup.name}>
+                        <Typography
+                            variant="body1"
+                            color={permanentLimitErrorMessage ? red[500] : undefined}
+                            sx={{
+                                maxWidth: '100px',
```

Comment:
```suggestion
                                maxWidth: '10ch',
```
just fyi, if you want ten characters instead of one hundred pixel, you can do this instead (it's not very clear in the us)

Reply:
Nice. Done here : https://github.com/gridsuite/gridstudy-app/pull/3427/commits/eb56dd878c077e31d0d61f8ed5fa41843e4ced61